src/components/fx/hover.js

```diff
@@ -889,6 +889,15 @@ function createHoverText(hoverData, opts) {
     var ya = c0.ya;
     var axLetter = hovermode.charAt(0);
     var t0 = c0[axLetter + 'Label'];
+
+    // search in array for the label
+    if(t0 === undefined && xa.type === 'multicategory') {
+        for(var q = 0; q < hoverData.length; q++) {
+            t0 = hoverData[q][axLetter + 'Label'];
```

Comment:
Almost there :)
To improve performance a bit, I suggest we add the line `var axLabel = axLetter + 'Label';` after `var axLetter = hovermode.charAt(0);` on line 890 and use this new variable here and when setting `var t0 = ...` on line 891.
Thank you!